frappe/core/doctype/system_settings/system_settings.json

```diff
@@ -528,12 +529,19 @@
    "fieldtype": "Select",
    "label": "Rounding Method",
    "options": "Banker's Rounding (legacy)\nBanker's Rounding\nCommercial Rounding"
+  },
+  {
+   "default": "0",
+   "description": "Strictly follow configured permissions",
+   "fieldname": "disable_sharing",
```

Comment:
```suggestion
   "fieldname": "disable_access_sharing",
```

or

```suggestion
   "fieldname": "disable_permission_sharing",
```

to avoid future conflict with other sharing features?

Reply:
I think `disable_document_sharing` would be more explicit (/apt) in this case.

Reply:
Are we sharing the document or the permission to access it? 😁

Reply:
Since it's referred in the docs as document sharing in multiple places, that seems like the best option. Users are not familiar with this feature as access/permission sharing. `DocShare` also gives that impression 😄
<img width="700" alt="Screenshot 2023-03-14 at 3 42 38 PM" src="https://user-images.githubusercontent.com/25857446/224968810-c96d0019-6546-4925-9545-23c761dc77ca.png">

<img width="500" alt="Screenshot 2023-03-14 at 3 44 54 PM" src="https://user-images.githubusercontent.com/25857446/224969188-15227621-7029-4f88-a502-1627dc7af3fd.png">